usagov/usagov-2021
# Usagov 2146 phsptan fixes

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-2146

## Description

Re-factors existing code (works the same) to eliminate phsptan errors.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps

Confirm functionality is unaffected: 

- Wizard breadcrumbs display unchanged
- Wizard mobile menu displays the same.
- Form still loads at /admin/config/usagov_ssg_postprocessing/toggle_tome
- Page still loads at /static-site-status
- Page still loads at /user/login and users can log in
- Form loads at /admin/config/usagov_directories/add_language_toggles
- Form loads at /admin/config/usagov_directories/add_synonyms
- Form loads at /admin/config/usagov_directories/add_acronyms
- Fixed errors in usagov workflow module but it is disabled.
- Page loads at /admin/config/usa_admin_styles/main
- JSON loads at /jsonapi/archived-nodes and /jsonapi/deleted-nodes
- Form works at /admin/config/content/orphaned-entities

Running phpstan locally should show no errors.
PHPStan invoked by CircleCi scan should show no errors.

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
